app/controllers/record_controller.rb

```diff
@@ -22,6 +23,23 @@ def record
     render 'record'
   end
 
+  def direct_link
+    fetch_eds_record
+    redirect_to @record.fulltext_link[:url]
+  end
+
+  private
+
+  def valid_url!
```

Comment:
Is this stylistically better than the `return unless` construction, or is it just better in this context because it matches with the other `before_action`?

Reply:
Using before filters, we are running the `valid_url!` check for every route in this controller (because we don't declare which routes to run it on). The previous `return unless` was fine as it was only needed in one route, but once I added a second route, DRYing it up seemed better.

Reply:
👍